src/components/waiting-list/ReservationEntry.js

```diff
@@ -1,35 +1,34 @@
-import React from 'react';
-
-import DefaultButton from '../buttons/DefaultButton';
-import './ReservationEntry.css';
+import React from "react";
 
+import DefaultButton from "../buttons/DefaultButton";
+import ReservationMenu from "./ReservationMenu";
+import "./ReservationEntry.css";
 
 class ReservationEntry extends React.Component {
   constructor(props) {
     super(props);
-    this.name = props.name;
-    this.people = props.people;
-    this.phone = props.phone;
   }
 
   render() {
+    const { name, people, phone, ...menuProps } = this.props;
```

Comment:
Is this a common pattern/convention? Have seen similar deconstructs 🤔

Reply:
Yes, it is an easy way to avoid sending certain props to the children components.
You can check this common patterns: https://reactpatterns.com/#jsx-spread-attributes